src/click.js

```diff
@@ -63,7 +64,10 @@ function clickElement(element, init, {clickCount}) {
     const shouldFocus = element.ownerDocument.activeElement !== element
     if (continueDefaultHandling) {
       if (previousElement) blur(previousElement, init)
-      if (shouldFocus) focus(element, init)
+      if (shouldFocus) {
+        if (element.tagName !== 'DIV' || isContentEditable(element))
+          focus(element, init)
+      }
```

Comment:
I don't think we need this change because the `focus` utility will exit early if the given element is not focusable. 

```suggestion
      if (shouldFocus) focus(element, init)
```

Reply:
I have tried on the div without the content editable but the focus was trigfered. I'll take a look again

Reply:
The problem here was the a div is not in the list of focusable elements. I have moved the check here https://github.com/testing-library/user-event/blob/2335842d247a6ae2e761f96f81e99fd1628e0a3f/src/utils.js#L167-L172.

Reply:
Let's change `isFocusable` to support this use case so we don't have to make this block of code more complex.